internal/cli/atlas/privateendpoints/gcp/create.go

```diff
@@ -43,7 +43,7 @@ func (opts *CreateOpts) initStore(ctx context.Context) func() error {
 	}
 }
 
-var createTemplate = "GCP private endpoint '{{.ID}}' created.\n"
+var createTemplate = "GCP private endpoint '{{.Id}}' created.\n"
```

Comment:
it doesn't appear as if we include the cloud provider in the message for the other providers. we should probably omit it here to remain consistent.
```suggestion
var createTemplate = "Private endpoint '{{.Id}}' created.\n"
```